main/src/ca/uwaterloo/flix/language/phase/jvm/BackendType.scala

```diff
@@ -178,6 +178,7 @@ object BackendType {
       case MonoType.String => BackendObjType.String.toTpe
       case MonoType.Regex => BackendObjType.Regex.toTpe
       case MonoType.Region => BackendObjType.Region.toTpe
+      case MonoType.Null => BackendObjType.Native(JvmName.ofClass(classOf[Object])).toTpe // to check
```

Comment:
@JonathanStarup can you verify the JVM stuff?

Reply:
looks good to me - the whole PR